components/dashboard/src/settings/Integrations.tsx

```diff
@@ -395,7 +395,7 @@ function GitIntegrations() {
             <div className="w-full flex h-80 mt-2 rounded-xl bg-gray-100 dark:bg-gray-900">
                 <div className="m-auto text-center">
                     <h3 className="self-center text-gray-500 dark:text-gray-400 mb-4">No Git Integrations</h3>
-                    <div className="text-gray-500 mb-6">In addition to the default Git Providers you can authorize<br /> with a self hosted instace of a provider.</div>
+                    <div className="text-gray-500 mb-6">In addition to the default Git Providers you can authorize<br /> with a self-hosted instace of a provider.</div>
                     <button className="self-center" onClick={() => setModal({ mode: "new" })}>New Integration</button>
```

Comment:
**issue**: I'm still getting the same behavior when trying to register a GH integration with GitHub.com. Maybe I'm doing something wrong. Pasting below some screenshots in case these help. 😭 

| OAuth Application | Modal |
|-|-|
| ![image](https://user-images.githubusercontent.com/120486/115383311-56712780-a1de-11eb-9c23-b2af2312d4f2.png) | ![image](https://user-images.githubusercontent.com/120486/115383314-57a25480-a1de-11eb-8650-7d8784d62028.png) |

Reply:
Is this screenshot of the modal from `/integrations` or from the initial setup. 
From the logs, it looks like you went through the setup and created the gitlab.com integration. Then you went to the /integrations and tried to setup github.com.

I cannot find any error in the logs. Actually, the github.com integration seems to be ok. Perhaps it's an issue with a missing update in the frontend. 

Let me check this in a new deployment. 

BTW, I always tested setup with both providers by deleting the account in between, to make sure it works both ways. Integrations page has some differences in behavior. Testing this is as well is great 👍🏻

Reply:
This is the screenshot of the modal after clicking Activate Integration and being redirected back to the modal with the warning message at the top. See also https://github.com/gitpod-io/gitpod/pull/3995#issuecomment-822526861.

> From the logs, it looks like you went through the setup and created the gitlab.com integration. Then you went to the /integrations and tried to setup github.com.

Exactly!

> BTW, I always tested setup with both providers by deleting the account in between, to make sure it works both ways. Integrations page has some differences in behavior. 

💡

Reply:
**issue:** Also, regardless if the Git Integration failed I can see the Git Provider added in the Git Providers list and I'm also able to connect. Expected?

![image](https://user-images.githubusercontent.com/120486/115387482-6d664880-a1e3-11eb-886d-5feafee1cc58.png)

Reply:
@gtsiolis, yes it is expected if you created the GH integration on the /integrations page ☝🏻 

From there, you can select to edit the GH integration and make it work eventually.

Reply:
> This is the screenshot of the modal after clicking Activate Integration and being redirected back to the modal with the warning message at the top. See also #3995 (comment).

>> From the logs, it looks like you went through the setup and created the gitlab.com integration. Then you went to the /integrations and tried to setup github.com.

> Exactly!

Interesting, it seems to be an issue with the modal rendered on the /integrations page then! Thanks, will check.